content/docs/command-reference/artifacts/get.md

```diff
@@ -48,14 +36,13 @@ of the DVC repository. Artifacts declared in other `dvc.yaml` files should be
 addressed in the form `path/to/dvc.yaml:artifact_name` or
 `path/to:artifact_name` (where `dvc.yaml` is omitted).
 
-<admon type="info">
-
-`dvc artifacts get` will first try to download artifacts via the <abbr>model
-registry</abbr>. If you do not have a valid Studio token, or the artifact is not
-tracked in the model registry, DVC will fall back to downloading the artifact
-from the project's default DVC remote.
-
-</admon>
+`dvc artifacts get` will first try to download artifacts via the Studio [REST
+API]. Downloading an artifact using the Studio REST API only requires a valid
+Studio [access token] and a Studio project configured with your [remote storage
+credentials]. It does not require the client to have those credentials. If you
+do not have a valid Studio token, or the artifact is not tracked in the model
+registry, DVC will fall back to downloading the artifact from the project's
```

Comment:
minor: it is in either way is downloaded from the remote. I'm not it will be clear to users what is going on here.

Reply:
How about "DVC will fall back to using the project's default DVC remote config to access remote storage"?

Reply:
okay, may be I missing something then.

does it go into `dvc get` mode here? cloning the repo, etc ... it requires access to GH then, etc?

even to take a step back ... can you run `dvc artifacts get` outside a repository? when the only thing that is set is an env var with a studio token? 

Should it be like:

`dvc artifacts get`:

- not need for a repo
- no GH access needed
- no AWS/GCP/Azure access needed
- no need to know that path, git hashes, other low level details
- unfortunately DVC is still needed to be installed (heavy, etc)
- can be run as simple as: `dvc artifacts get model_name`

If not studio token or artifact is not tracked in the model registry (what happens if there are not credentials in Studio btw?): it falls back to what exactly, could you clarify please?

- extracts the path to an artifact from the current repo / current revision? (means it needs a repo for example)
- fails if it runs outside the repo
- etc ...

Reply:
> does it go into `dvc get` mode here? cloning the repo, etc ... it requires access to GH then, etc?

Yes.

> even to take a step back ... can you run `dvc artifacts get` outside a repository? when the only thing that is set is an env var with a studio token?

Yes.

There is no need to be in a repo (same is true for `dvc get`) regardless of the "mode" `dvc artifacts get` uses.

Another alternative to explain here is to link to `dvc get` and say something like "DVC will fall back to its typical method to get files (see `dvc get`)."

Reply:
If there is no token set, how does it resolve model name into a repo + a path? Or does it expect those to be provided?

Reply:
Repo is a required arg, and it uses GTO to find the right path and rev.

Reply:
> Another alternative to explain here is to link to `dvc get` and say something like "DVC will fall back to its typical method to get files (see `dvc get`)."

Pushed this change to the text.

Reply:
Sounds good, Dave.

> Repo is a required arg, and it uses GTO to find the right path and rev.

As a followup it would great to make it optional for the `artifacts get`. Probably we were discussing this before ( I don't remember the details), but there is not really a reason for `artifacts get` being complicated, wdyt?

Reply:
Should we open a follow-up issue in studio to discuss adding it to the rest api?

Reply:
why rest API though? would we need it first to be implemented there? ... yes, overall, let's do a follow up on this

Reply:
I think I'm misunderstanding then. Do you mean if I'm already inside a repo, we should default to that repo?

Reply:
No, I mean no matter if I'm inside or outside, I assume that in a lot of case mode name is unique across many repos, there is no need further clarify that if Studio has a single model with that name. In some cases clarifying it's subdir should be enough (repo is not needed). In some cases repo is needed indeed.

Reply:
> Do you mean if I'm already inside a repo, we should default to that repo?

Can be, I would for simplicity make independent on anything local.